dev/com.ibm.ws.jmx_fat/override/autoFVT/src/ant/launch.xml

```diff
@@ -24,7 +24,22 @@
 		</not>
 	</condition>
 	<property name="illegal.access.permit.jmx.fat" value=""/>
-		
+	
+	<!-- Java 17 began the security manager's deprecation process (https://openjdk.java.net/jeps/411).
+		     In Java 18, the `java.security.manager` system property's default changed from `allow` to `disallow`, which causes our build process to fail.  
+			 Setting up a gated property (use.java.security.manager) so that any builds that take place on Java 15+ use the JVM override 
+			 -Djava.security.manager=allow which is needed for our build process, but older versions will be skipped --> 
+	<condition property="use.java.security.manager" value="-Djava.security.manager=allow">
+		<not>
+			<or>
+				<equals arg1="11" arg2="${java.specification.version}"/>
+				<equals arg1="1.7" arg2="${java.specification.version}"/>
```

Comment:
We don't support Java 7 any longer, so it isn't needed here.

Reply:
Thanks.  I have removed the 1.7 references from OL.

Reply:
It is still here.

Reply:
I put them back after we talked, but I have removed them again.